packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -83,6 +83,10 @@ function CreateIndexModal({
   const enableInIndexesGuidanceExp = usePreference('enableIndexesGuidanceExp');
   const showIndexesGuidanceVariant =
     usePreference('showIndexesGuidanceVariant') && enableInIndexesGuidanceExp;
+  if (showIndexesGuidanceVariant && !currentTab) {
+    // If user is in the variant and there has not been a currentTab set, default it to IndexFlow
+    props.onTabClick('IndexFlow');
```

Comment:
I think we should avoid setting something like this in the react flow. We're only doing this for telemetry purposes right? Can we instead set that telemetry event metadata based on the preferences? And then we can go back to always having the currentTab set.